### DISCUSSION TITLE
Clarification on organ settings

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1731] [General]
In the organ settings page may I have clarification ..

Amplitude : increase sound level and give it more louder ?

Gain : increase sound level

Manual tuning : Tune stops
Eg : -1200 to go 1 octave lower ?

Auto Tuning correction : ??

Mécanique (ms) : ??

Release tail length : ??

Thank you very much for your explanations ...

**Comment by oleg68** [comment]
I suggest you to press on the `Help` button and read more information.
